Add source and versioninfo to exercise config (#1149)
exercism/v3: languages/csharp/reference/examples/new-concept-exercise-arrays.md

## Step 10: add .meta/config.json:

> This file contains meta information on the exercise, which currently includes the exercise's contributors, and language version requirements ([example][config.json-example]).

Reviewer: ```suggestion
This file contains meta information on the exercise, which currently only includes the exercise's contributors ([example][config.json-example]).
```
